documentation/docker.asciidoc

```diff
@@ -36,10 +38,42 @@ This can be done by your projects ide-admin in the link:usage.asciidoc#developer
 "DOCKER_EDITION=DOCKERDESKTOP" 
 (without the quotes around the allocation) to the file devon.properties located at the path ${DEVON_IDE_HOME}/conf/devon.properties.
 
-After successfull installation please start Rancher Desktop and go to the menu point "Kubernetes Settings". Under "Container Runtime" choose dockerd(moby) to enable the docker-cli. If you use Windows, then make sure to go to the Settings named "WSL Integration".
-There you can choose the WSL Shells you want Rancher Desktop to work on. Besides that the tools like docker or kubectl should 
+=== installation and setup of Rancher Desktop 
+
+===== installation
+
+The installation of Rancher Desktop will be handled with the help of a graphical user interface. After starting the installation process from your shell you may see a small progress window for a few seconds before the installer starts. 
+
+image::images/RDInstallerInfoWindow.PNG[]
```

Comment:
sorry, for being picky, but can you consistently use lower-case extensions like in `EnablingDocker.png` (instead of `.PNG`).

Reply:
Yes, I'll do so and I also welcome this feedback regarding consistency.